packages/rpc/src/util.ts

```diff
@@ -1,43 +1,24 @@
+import process from 'node:process';
+
+// eslint-disable-next-line import/no-mutable-exports
 let register: (id: string) => void = () => false;
 try {
-	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+	// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
 	const { app } = require('electron');
-	// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-assignment
 	register = app.setAsDefaultProtocolClient.bind(app);
-} catch (err) {
+} catch {
 	try {
 		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-require-imports
 		register = require('register-scheme');
-	} catch (e) {} // eslint-disable-line no-empty
+	} catch {} // eslint-disable-line no-empty
```

Comment:
I believe this eslint-disable comment is not necessary
```suggestion
	} catch {}
```

Reply:
Nope, it is a problem
<img width="226" alt="Screenshot 2022-10-03 at 00 50 38" src="https://user-images.githubusercontent.com/46065349/193468532-225fa763-93bc-4535-8607-129f14504945.png">